util/argParser.js

```diff
@@ -250,6 +250,13 @@ class ArgParser {
         type: "number",
       },
 
+      "dedupPolicy": {
+        describe: "Deduplication policy",
+        default: "skip",
+        type: "string",
+        choices: ["skip", "revisit", "keep"],
```

Comment:
Might not be a bad idea to add some input validation here so that if a user adds an incorrect value it'll default back to `skip` rather than throwing an error. Otherwise looks great, thank you!

Reply:
Going to add in a follow-up commit after merging :)###

Reply:
yargs will validate the `choices` list so should be good here!